AppHandling/Publish-NavContainerApp.ps1

```diff
@@ -1,3 +1,21 @@
+$sslCallbackCode = @"
+	using System.Net.Security;
+	using System.Security.Cryptography.X509Certificates;
+
+	public static class SslVerification
+	{

```

Comment:
Why would you need this here (as well as in HelperFunctions)
The variables and types from HelperFunctions should always be loaded when running.

Reply:
then it is indeed unneeded - sorted :)